### DISCUSSION TITLE
Unable to generate mock files using Cuckoo 2.0.9 in SPM

### DISCUSSION THREAD

**Discussion by alizainc** [Brightify/Cuckoo#505] [Q&A]
I have been trying to create the GenerateMocks.swift file using Cuckoo 2.0.9 in SPM but I do not see any files being created. I have the Cuckoo and CuckooGenerator libs linked to the test target and CuckooPluginSingleFile in the Build Tools Plugin. I have also created the Cuckoofile.toml file but when I do `Product > Run` to does not create any mock files.

<img width="960" alt="Screenshot 2024-07-23 at 12 45 56" src="https://github.com/user-attachments/assets/f7350c02-78bd-4370-8272-c6e3098815ce">

```
# output = "./MacClientManagerTests/GeneratedMocks.swift"

[modules.MacClientManager]
imports = ["Foundation"]
testableImports = ["MacClientManager"]
sources = [
    "./MacClientManager/ViewControllers/StartAppUrlViewController.swift",
]
exclude = []
# regex = ""
output = "./MacClientManagerTests/GeneratedMocks.swift"

[modules.MacClientManager.options]
glob = true
keepDocumentation = true

# If specified, Cuckoo can also get sources for the module from an Xcode target.
[modules.MacClientManager.xcodeproj]
# Path to folder with .xcodeproj, omit this if it's at the same level as Cuckoofile.
path = "."
target = "MacClientManager"

```

Can you help guide why I am unable to generate the mock file?

**Comment by ubitCloud** [comment]
@alizainc Please check my demo project where I can use cuckoo via SPM.
https://github.com/ubitCloud/CuckooP3
